test/ProgressOnderwijsUtils.Tests/PropertyMappingTest.cs

```diff
@@ -60,7 +62,40 @@ public void MapProperties_werkt_voor_single_mapper()
         }
 
         [Fact]
-        public void MapProperties_werkt_niet_voor_multiple_mapper()
+        public void MapProperties_Identity_map_doet_niks()

```

Comment:
waarom dit in t nl, rest van deze lib is engels toch? (volgende test ook)

Reply:
omdat de eerdere testen ook in nl waren, maar blijkbaar is dat gemengd.

We kunnen wel wat hernoemen, bijv. gewoon hier in github (immers is dit een C# symbol soort die betrouwbaar 0 refs heeft ;-) )